js/app.js

```diff
@@ -95,7 +115,7 @@ TextApp.prototype.onSettingsReady_ = function() {
   this.analytics_.setEnabled(this.settings_.get('analytics'));
```

Comment:
the analytics and sindowController stuff doesn't seem editor related, should that remain in onSettingsReady?

Reply:
Very fair point, will move, thanks!

Reply:
Yep good idea